src/Array.mo

```diff
@@ -160,6 +160,12 @@ module {
     };
     return null
   };
+  public func exists<X>(array : [X], predicate : X -> Bool) : Bool {
```

Comment:
This is called `some` in Trie.mo and List.mo. I'd hate to introduce inconsistency. Does the bot insist one this?

Reply:
Also would need doc and some tests, ideally.